api/tbcapi/README.md

```diff
@@ -101,113 +101,154 @@ The `tbcd` daemon runs an RPC server that listens on the address provided by the
 You can run the `tbcd` daemon with the RPC server enabled with the following command:
 
 ```shell
+# Make sure to replace localhost:8082 with your desired address and port
 TBC_ADDRESS=localhost:8082 /path/to/tbcd
 ```
 
-### 👉 RPC Client
+### 🔧 RPC Client
 
 [`hemictl`](../../cmd/hemictl) serves as a reference implementation of an RPC client tailored for interacting
-with the `tbcd` daemon.
+with the `tbcd` daemon. The client provides a command-line interface for all supported RPC commands.
 
 ---
 
 ## 📚 Resources
 
-For developers looking to integrate or extend functionality, view the raw Go types used in TBC's RPC commands:
-[View `tbcapi.go`](tbcapi.go).
+For developers looking to integrate or extend functionality:
+- View the raw Go types used in TBC's RPC commands: [View `tbcapi.go`](tbcapi.go)
+- Check our [contribution guidelines](../../CONTRIBUTING.md)
+- Review our [security policy](../../SECURITY.md)
 
 ---
 
 ## 📡 Protocol
 
-The **RPC protocol** is WebSocket-based and follows a standard request/response model. For more detailed information,
+The **RPC protocol** is WebSocket-based and follows a standard request/response model. All communications are done via JSON messages. For more detailed information,
 refer to the [protocol documentation](../protocol/README.md).
 
 ### 🚫 Errors
 
 If an error occurs during a request, the response payload will include an `error` value containing the following
 details:
 
-| Field       | Description                                                                          |
-|-------------|--------------------------------------------------------------------------------------|
-| `timestamp` | The time at which the error occurred, in Unix seconds.                               |
-| `trace`     | A unique string for tracing errors between server and client (internal errors only). |
-| `message`   | The error message. For internal server errors, this will read `internal error`.      |
+| Field       | Description                                                                          | Example |
+|-------------|--------------------------------------------------------------------------------------|---------|
+| `timestamp` | The time at which the error occurred, in Unix seconds.                               | `1679198400` |
+| `trace`     | A unique string for tracing errors between server and client (internal errors only). | `err_xyz123` |
```

Comment:
The `trace` in errors is usually 8 random bytes, encoded as a hexadecimal string ([`github.com/hemilabs/heminetwork/api/protocol.Errorf`](https://github.com/hemilabs/heminetwork/blob/316d517ab2aa4f8931caf958fe78a84e128d58db/api/protocol/protocol.go#L187-L195)).

I think providing an example in this format, whilst it doesn't matter too much, could prevent any confusion with the trace string:
```suggestion
| `trace`     | A unique string for tracing errors between server and client (internal errors only). | `804d952f893e686c` |
```